CVLByExample/ReadOnlyReentrancy/contracts/VulnerableBankOther.sol

```diff
@@ -0,0 +1,30 @@
+import "@openzeppelin/contracts/utils/math/SafeMath.sol";
+
+contract VulnerableBankOther {
+  
+    using SafeMath for uint256;
```

Comment:
This is the main difference from BulnerableBank.sol?

Reply:
yeah not really a lot of difference